test/pattern/dummy.c

```diff
@@ -259,3 +262,8 @@ void mutt_buffer_select_file(struct Buffer *file, SelectFileFlags flags,
                              char ***files, int *numfiles)
 {
 }
+
+struct Mailbox *ctx_mailbox(struct Context *ctx)
+{
+  return Context ? Context->mailbox : NULL;
+}
```

Comment:
Please can you add a Doxygen function comment block (for complete-ness).
It doesn't need to say much.

Reply:
Mon Dec 21 13:37:12 GMT 2020

Done - see a12f1ff.